lib/modules/datasource/types.ts

```diff
@@ -90,6 +90,12 @@ export interface ReleaseResult {
   packageScope?: string;
 }
 
+export interface PostprocessReleaseConfig {
+  datasource?: string;
```

Comment:
we should have two interfaces, a public with datasource and a private without. otherwise someone could expect the datasource property is passed through.